CODEOWNERS.md

```diff
@@ -0,0 +1 @@
+* @baloise/open-source
```

Comment:
Rather than using the group I'd recommend adding the actual project maintainer (github handles) here.

Reply:
done in cea0962a011f78c31fb23e5f4a246a514604bda6